modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc

```diff
@@ -6,7 +6,7 @@ endif::[]
 
 Support Category: Premium
 
-SAP S/4HANA Cloud Connector v1.0
+SAP S/4HANA Cloud Connector v1.1
 
 The Mule SAP S/4HANA Cloud Connector provides full support for the query, get, create, update, delete entities, 
```

Comment:
```suggestion
Anypoint Connector for SAP S/4HANA Cloud (SAP S/4HANA Cloud Connector) provides full support for the query, get, create, update, and delete entities, 
```

Reply:
Based on what the following text means, perhaps this might be better:
```suggestion
Anypoint Connector for SAP S/4HANA Cloud (SAP S/4HANA Cloud Connector) provides full support for execute functions, as well as for the query, get, create, update, and delete entities, using the ODATA API exposed by S/4HANA.
```

Reply:
Or...:
```suggestion
Using the ODATA API exposed by S/4HANA, Anypoint Connector for SAP S/4HANA Cloud (SAP S/4HANA Cloud Connector) provides full support for....
```